manifests/bosh-manifest/operations.d/030-set-stemcell.yml

```diff
@@ -2,5 +2,5 @@
 - type: replace
   path: /resource_pools/name=vms/stemcell
   value:
-    url: https://bosh.io/d/stemcells/bosh-aws-xen-hvm-ubuntu-bionic-go_agent?v=1.91
-    sha1: 17e8282e1366ca1acfc60c24ffb23b1ed5147407
+    url: https://bosh.io/d/stemcells/bosh-aws-xen-hvm-ubuntu-bionic-go_agent?v=1.97
+    sha1: sha1:c1f4202e0b343d8e8f2bc218e735ada87f85b93b
```

Comment:
```suggestion
    sha1: c1f4202e0b343d8e8f2bc218e735ada87f85b93b
```

Reply:
Now I'm wondering what difference this ops file makes 🤔 
I can see all the vms are on 'bosh-aws-xen-hvm-ubuntu-bionic-go_agent/1.97'